numpy/core/shape_base.py

```diff
@@ -275,3 +278,67 @@ def hstack(tup):
         return _nx.concatenate(arrs, 0)
     else:
         return _nx.concatenate(arrs, 1)
+
+def stack(arrays, axis=0):
+    """
+    Join a sequence of arrays along a new axis.
```

Comment:
How about
```
def stack(arrays, axis=0):
    """
    Join a sequence of arrays along a new axis.

    The position of the new axis in the stacked result is specified by the
    `axis` parameter. For instance, if ``axis=-1`` the result will have
    been stacked along the last axis, and if ``axis=0`` it will have been
    stacked along the first axis.

    etc...
```